test/load-generator/state.go

```diff
@@ -183,7 +183,9 @@ type State struct {
 
 	challSrv    *challtestsrv.ChallSrv
 	callLatency latencyWriter
-	client      *http.Client
+
+	directory  *acme.Directory
```

Comment:
For 'realism' sake perhaps this should be a field on `account` rather than the overall state, so each new client interaction requires pulling the directory. Seems unlikely we'll have load issues with that, but who knows. It might be a bit of a pain to implement though...

Reply:
Ehh... I had implemented the directory initially in a way that let it be "refreshed" from the server but it meant I had to add a mutex to the map and it didn't seem like it was worthwhile. I still think it isn't worthwhile to drive traffic to the directory endpoint. If that's our goal I'd rather just make plain GET requests and track the latency and not bother with parsing the endpoints/directory resource.

Reply:
:+1:, keeping as-is works for me.